lib/dialects/abstract/connection-manager.js

```diff
@@ -1,26 +1,18 @@
 'use strict';
 
-const Pooling = require('generic-pool');
+const Pool = require('sequelize-pool').Pool;
```

Comment:
I think you can use destructuring with node v6 eg. `const { Pool } = require('sequelize-pool')`

Reply:
:+1: